applications/luci-app-vnstat2/htdocs/luci-static/resources/view/vnstat2/config.js

```diff
@@ -0,0 +1,113 @@
+// This is free software, licensed under the Apache License, Version 2.0
+
+'use strict';
+'require fs';
+'require ui';
+'require uci';
+'require form';
+'require tools.widgets as widgets';
+
+return L.view.extend({
+	handleDeleteModal: function(m, iface, ev) {
+		L.showModal(_('Delete interface <em>%h</em>').format(iface), [
+			E('p', 'The interface will be removed from the database permanently. This cannot be undone.'),
+			E('div', { 'class': 'right' }, [
+				E('div', {
+					'class': 'btn',
+					'click': L.hideModal
+				}, _('Cancel')),
+				' ',
+				E('div', {
+					'class': 'btn cbi-button-negative',
+					'click': ui.createHandlerFn(this, 'handleDelete', m, iface)
+				}, _('Delete'))
+			])
+		]);
+	},
+
+	handleDelete: function(m, iface, ev) {
+		return fs.exec('/usr/bin/vnstat', ['--remove', '-i', iface, '--force'])
+			.then(L.bind(m.render, m))
+			.then(L.hideModal)
+			.catch(function(e) {
+				L.hideModal();
+				ui.addNotification(null, E('p', e.message));
+			});
+	},
+
+	render: function() {
+		var m, s, o;
+
+		m = new form.Map('vnstat', _('vnStat'), _('vnStat is a network traffic monitor for Linux that keeps a log of network traffic for the selected interface(s).'));
+
+		s = m.section(form.TypedSection, 'vnstat', _('Interfaces'));
+		s.anonymous = true;
+		s.addremove = false;
+
+		o = s.option(widgets.DeviceSelect, 'interface', _('Monitor interfaces'), _('The selected interfaces are automatically added to the vnStat database upon startup.'));
+		o.rmempty = true;
+		o.multiple = true;
+		o.noaliases = true;
+		o.nobridges = false;
+		o.noinactive = false;
+		o.nocreate = false;
+
+
+		o = s.option(form.DummyValue, '_database');
+
+		o.load = function(section_id) {
+			return Promise.all([
+				fs.exec('/usr/bin/vnstat', ['--json', 'f', '1']),
+				uci.load('vnstat')
```

Comment:
The `uci.load('vnstat')` should be redundant since `form.Map('vnstat', ...)` already takes care of loading it.